README.md

```diff
@@ -25,3 +25,13 @@ Link Visual Studio Code to that database by going to ```View``` and selecting th
 
 If all went correctly, your database should be connected, and you can then run the project.
 To run the project, press the green "play" triangle that is above the code editor, roughly in the middle.
+
+# Human Resources
+
+## Departments
+
+###View All
+The great folks in the Human Resources Department can view all Departments when the Department Naviagation Link is clicked.
+
+###Add A New Department

```

Comment:
I would save this for when the create link is functional, but it's fine for now.

Reply:
My tickets were for View all Departments and Create new departments. I accidentally described both in the readme when I should have only described only view all for this pull request.